metricbeat/module/postgresql/_meta/docker-entrypoint-initdb.d/0-enable-pg_stat_statements.sh

```diff
@@ -1,2 +1,2 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash
```

Comment:
It's not clear to me what changed here, I assume it's the line ending.

Reply:
Yeah, line endings.